xml/System.Net.Http/HttpClient.xml

```diff
@@ -455,7 +457,7 @@ The environment variables used for `DefaultProxy` initialization on Windows and
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Headers set on this property don't need to be set on request messages again.  
+ Headers set on this property don't need to be set on request messages again. `DefaultRequestHeaders` should not be modified after any request has been sent, because it is not thread-safe.
```

Comment:
I would word this better; it's fine to change default headers so long as there are no outstanding requests.

Reply:
An example of where you might do this is: perform some login request, set a default header with a returned auth token, and then party on.